app/adapters/crate.js

```diff
@@ -5,6 +5,15 @@ const BULK_REQUEST_GROUP_SIZE = 10;
 export default class CrateAdapter extends ApplicationAdapter {
   coalesceFindRequests = true;
 
+  findRecord(store, type, id, snapshot) {
+    let { include } = snapshot;
+    // This ensures `crate.versions` are always fetched from another request.
+    if (include === undefined) {
+      snapshot.include = 'keywords,categories,badges,downloads';
```

Comment:
yeah, exactly!

Reply:
> we might want to consider adding the `default_version` so that we don't need two requests for the common case of displaying the default version

Forgot to mention, I think this would be better done in follow-up PR, otherwise some assertion that expect versions are not loaded might failed.